exporter/dynatraceexporter/metrics_exporter_test.go

```diff
@@ -119,57 +126,97 @@ func Test_exporter_PushMetricsData(t *testing.T) {
 	doubleHistogramDataPoint := doubleHistogramDataPoints.AppendEmpty()
 	doubleHistogramDataPoint.SetCount(2)
 	doubleHistogramDataPoint.SetSum(10.1)
-	doubleHistogramDataPoint.SetTimestamp(pdata.Timestamp(100_000_000))
+	doubleHistogramDataPoint.SetExplicitBounds([]float64{0, 2, 4, 8, 16})
+	doubleHistogramDataPoint.SetBucketCounts([]uint64{0, 1, 0, 1, 0})
```

Comment:
Shouldn't the `number of bucket counts` be the `number of bounds` + 1? This is how it is in Java at least, are all elements in the first bucket lower the lowest bucket bound? Or all elements in the last bucket larger than the last?